src/aspire/utils/optimize.py

```diff
@@ -156,12 +163,14 @@ def identity(input_x):
             info.p.append(p)
 
         if cg_opt["verbose"]:
-            print('[CG] Initialized. Residual: {}. Objective: {}'.format(np.linalg.norm(info.res[0]), np.sum(info.obj[0])))
+            logger.info('[CG] Initialized. Residual: {}. Objective: {}'.format(np.linalg.norm(info.res[0]),
+                                                                              np.sum(info.obj[0])))
 
         if np.all(res < b_norm * cg_opt["rel_tolerance"]):
             break
 
     if i == cg_opt["max_iter"] - 1:
-        raise Warning('Conjugate gradient reached maximum number of iterations!')
+        logger.warning('Conjugate gradient reached maximum number of iterations!')
```

Comment:
Add `[CG]` here like the other log messages?

Reply:
Done.